### PR TITLE
NOOS-423/fix-firefox-installation

### PR DESCRIPTION
**Pull request by filiperosa**
Install firefox .deb package instead of snap

### CODE REVIEW

File: docker/circleci/Dockerfile [hunk 1]
```diff
@@ -6,13 +6,16 @@ FROM cimg/python:3.8.17
...
 # # ----------------------------
-ARG FIREFOX_VERSION="v0.30.0"
+ARG GECKODRIVER_VERSION="v0.33.0"
 RUN apt-get update && \
-    apt-get install -y firefox && \
-    apt-get clean && \
-    curl -sSL "https://github.com/mozilla/geckodriver/releases/download/${FIREFOX_VERSION}/geckodriver-${FIREFOX_VERSION}-linux64.tar.gz" | \
-    tar -xz -C /usr/local/bin
+    add-apt-repository ppa:mozillateam/ppa -y && \
+    echo -e '\nPackage: *\nPin: release o=LP-PPA-mozillateam\nPin-Priority: 1001\n' | tee /etc/apt/preferences.d/mozilla-firefox && \
```
Comment (pieeri) [inline docker/circleci/Dockerfile:15]: 🐼 

Maybe easier to maintain the APT control file manually rather through a linux pipe?

```sh
COPY mozilla-firefox /etc/apt/preferences.d/
RUN apt-get update && \
    add-apt-repository ppa:mozillateam/ppa -y && \
    apt install firefox && \
    ...
```

**Review by pieeri — CHANGES_REQUESTED** [review]
Small suggestion for better readability & maintainability.

File: docker/circleci/Dockerfile (hunk 1 above)
Comment (filiperosa) [reply docker/circleci/Dockerfile]: Good idea, done!

**Review by pieeri — APPROVED** [review]
Nice one! Your first Docker image out there in the wild!

**Pull request merged**